content/en/docs/installation/helm.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Helm"
+linkTitle: "Helm"
+weight: 22
+type: "docs"
+---
+
+> Note: From cert-manager `v1.2.0` onward, the minimum supported version of
+> Kubernetes is `v1.16.0`. Users still running Kubernetes `v1.15` or below should
+> upgrade to a supported version before installing cert-manager.
+
+> **Warning**: You should not install multiple instances of cert-manager on a single
+> cluster. This will lead to undefined behavior and you may be banned from
+> providers such as Let's Encrypt.
+
+## Installing with Helm
+
+> **Note**: cert-manager should never be embedded as a sub-chart into other Helm charts.
+> cert-manager manages non-namespaced resources in your cluster and should only be installed once.
+
+### Prerequisites
+
+- Helm v3 installed
+
+### Steps
+
+In order to install the Helm chart, you must follow these steps:
+
+#### 1. Add the Jetstack Helm repository:
+
+> **Warning**: It is important that this repository is used to install
+> cert-manager. The version residing in the [Helm stable repository](https://github.com/helm/charts/tree/master/stable/cert-manager) is
+> *deprecated* and should *NOT* be used.
+
+```bash
+$ helm repo add jetstack https://charts.jetstack.io
+```
+
+#### 2. Update your local Helm chart repository cache:
+
+```bash
+$ helm repo update
+```
+
+#### 3. Install `CustomResourceDefinitions`
+
+cert-manager requires a number of CRD resources to be installed into your
+cluster as part of installation.
+
+This can either be done manually, using `kubectl`, or using the `installCRDs`
+option when installing the Helm chart.
+
+> **Note**: If you're using a `helm` version based on Kubernetes `v1.18` or below (Helm `v3.2`) `installCRDs` will not work with cert-manager `v0.16`.
+> For more info see the [v0.16 upgrade notes](../upgrading/upgrading-0.15-0.16/#helm)
+
+##### Option 1: installing CRDs with `kubectl`
+
+Install the `CustomResourceDefinition` resources using `kubectl`:
```

Comment:
suggestion: this text doesn't provide any new information and I think it can be removed

```suggestion
```